.travis.yml

```diff
@@ -2,20 +2,24 @@ language: java
 addons:
   apt:
     sources:
```

Comment:
You're right that changing spacing in a PR is confusing. I've changed it back in the latest update.